site/docs/red-team/plugins/index.md

```
@@ -53,7 +53,7 @@ Click on a plugin to see its documentation.
 
 <PluginTable shouldGroupByCategory showRemoteStatus />
 
-_🌐 indicates that plugin uses remote inference_
+_🌐 indicates that plugin uses remote inference in Promptfoo Community edition_
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)